pkgs/applications/blockchains/exodus/default.nix

```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
 
   sourceRoot = ".";
   unpackCmd = ''
-      ${unzip}/bin/unzip "$src" -x "Exodus*/lib*so"
+      ${unzip}/bin/unzip "$src"
```

Comment:
Shouldn't this be auto extracted now?

Reply:
> Shouldn't this be auto extracted now?

All fixed up now. Looks like ofborg had an issue testing it.